include/benchmark/benchmark.h

```diff
@@ -1118,6 +1122,8 @@ class Benchmark {
   std::vector<Statistics> statistics_;
   std::vector<int> thread_counts_;
 
+  static TimeUnit default_time_unit_;
```

Comment:
So what's the default default here? It just gets zero initialised, but isn't set if the parameter is left empty?